app/src/main/java/com/bootcamp_android/parking_app/fragments/ReservationListFragment.kt

```diff
@@ -13,7 +15,15 @@ import com.bootcamp_android.parking_app.adapter.Adapter
 import com.bootcamp_android.parking_app.adapter.Provider.Companion.reservations
 import com.bootcamp_android.parking_app.databinding.FragmentReservationsBinding
 
-class ReservationListFragment : Fragment(R.layout.fragment_reservations) {
```

Comment:
Try to declare the binding with lateinit and clean it in onDestroy
Please do it as the following example https://github.com/android/architecture-components-samples/blob/main/ViewBindingSample/app/src/main/java/com/android/example/viewbindingsample/InflateFragment.kt

The idea is that you can use that binding variable in other methods